lib/executor.js

```diff
@@ -230,13 +230,19 @@ function runScripts(app, list, callback) {
   var functions = [];
   list.forEach(function(filepath) {
     debug('Requiring script %s', filepath);
-    var exports = require(filepath);
-    if (typeof exports === 'function') {
-      debug('Exported function detected %s', filepath);
-      functions.push({
-        path: filepath,
-        func: exports
-      });
+    try {
+      var exports = require(filepath);
+      if (typeof exports === 'function') {
+        debug('Exported function detected %s', filepath);
+        functions.push({
+          path: filepath,
+          func: exports
+        });
+      }
+    } catch (err) {
+      console.error('Failed loading boot script: %s\n%s',
+        path.basename(filepath), err.message);
```

Comment:
Why `path.basename` instead of the full path?

`err.message` is discarding the stack trace which usually contain a lot of useful information (IMO).

I am proposing the following:

```
console.error('Failed to load the boot script %s\n%s', filepath, err.stack)
```

Thoughts?